src/PerfView/EventViewer/EventWindow.xaml

```diff
@@ -242,6 +242,13 @@
                         MinWidth="60"
                         IsReadOnly="True" />
                         <WPFToolKit:DataGridTextColumn
+                        x:Name="TimeStampColumn"
+                        Header="TimeStamp"
+                        Binding="{Binding TimeStamp, StringFormat=u}"
```

Comment:
I would like to make sure that the column description contains the time zone - specifically either "UTC" or "Trace Local".  It looks like `u` is UTC, while `o` contains the time zone information.

Reply:
Sure, adding `Trace Local`